pgmpy/tests/test_independencies/test_Independencies.py

```diff
@@ -54,46 +54,21 @@ def tearDown(self):
         del self.assertion
 
 
-class IndependenciesTestCase(object):
-    def assertIndependenceAssertionEqual(self, assertion1, assertion2):
-        if not (assertion1.event1 == assertion2.event1):
-            raise AssertionError(str(assertion1.event1) + "is not equal to" + str(assertion2.event1))
-        if not (assertion1.event2 == assertion2.event2):
-            raise AssertionError(str(assertion1.event2) + "is not equal to" + str(assertion2.event2))
-        if not (assertion1.event3 == assertion2.event3):
-            raise AssertionError(str(assertion1.event3) + "is not equal to" + str(assertion2.event3))
-
-    def assertIndependenciesEqual(self, Independencies1, Independencies2):
-        if len(Independencies1) == len(Independencies2):
-            for map1, map2 in zip(Independencies1, Independencies2):
-                self.assertIndependenceAssertionEqual(map1, map2)
-
-
-class TestIndependencies(unittest.TestCase, IndependenciesTestCase):
+class TestIndependencies(unittest.TestCase):
```

Comment:
@kshitij10496 Can you add more test cases for `Independencies` class ? Test with all the possible input parameters for the methods.